### ISSUE TITLE
Web component: Leadspace block - Contact sale button is not operable with spacebar.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4612]
## Detailed description

Issue: Contact sale button is not operable with spacebar.

Expected Result:
Button should be operable with both enter key and spacebar.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/e7bc474d-a1e1-42ad-92fb-06b3f7c94d32)

**Comment by kennylam** [comment]
@Praveen-sr This is default browser/html behavior. Links (contact sales button) are activated with the 'enter' key. Spacebar by default will advance the page scroll.

cc: @RobertaJHahn
